README.md

```diff
@@ -3,6 +3,19 @@ KILDA CONTROLLER
 ---
 [![Build Status](https://github.com/telstra/open-kilda/actions/workflows/unittest.yml/badge.svg)](https://github.com/telstra/open-kilda/actions/workflows/unittest.yml)[![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=org.openkilda%3Akilda-parent&metric=alert_status)](https://sonarcloud.io/dashboard?id=org.openkilda%3Akilda-parent)
 
+## Description
+
+Kilda Controller is Software Defined Network Controller. It was created to manage set of switches, create connections 
+between them (flows) and react on network equipment failure by evacuation of flows from damaged points.
```

Comment:
I'd say "It was created to manage A set of switches, create connections between them (flows) and react TO network equipment failureS by EVACUATING flows from damaged points."

Reply:
fixed